src/CircularProgress/CircularProgress.js

```diff
@@ -186,13 +158,13 @@ const CircularProgress = React.createClass({
   _rotateWrapper(wrapper) {
     if (this.props.mode !== 'indeterminate') return;
 
-    autoPrefix.set(wrapper.style, 'transform', 'rotate(0deg)', this.state.muiTheme);
-    autoPrefix.set(wrapper.style, 'transitionDuration', '0ms', this.state.muiTheme);
+    autoPrefix.set(wrapper.style, 'transform', 'rotate(0deg)', this.context.muiTheme);
```

Comment:
We really need to fix this at some point. Passing theme as third parameter doesn't do anything anymore.